setup_template.py

```diff
@@ -36,31 +129,36 @@ def main():
         input("Enter a short description of the project: ")
         or "A beautiful description."
     )
+    repo_license = request_license()
 
     # Print the data
     print(
         f"Using the following values:\n"
         f"\tRepository name: '{repo_name}'\n"
         f"\tModule name: '{module_name}'\n"
         f"\tAuthor: '{username} <{email}'>\n"
-        f"\tDescription: '{description}'"
+        f"\tDescription: '{description}'\n"
+        f"\tLicense: '{repo_license.name if repo_license else 'Unlicensed'}'"
     )
     input("Press enter to continue...")
 
     # Replace the template values
-    for file in pl.Path(dir_repo).glob("**/*"):
+    for file in pl.Path(DIR_REPO).glob("**/*"):
         if (
             file.is_file()
-            and not file.name == file_self.name
+            and not file.name == __file__
```

Comment:
Pretty sure I tried that and it did not work reliably. (I think when working dir != project dir.

Reply:
Replaced with `setup_template.py`, no need to get fancy.